Update onboarding doc w/ lab meeting and Penn remainder (#30)
greenelab/onboarding: onboarding.md

We expect that lab members will keep agreements, be careful, and share their code and results openly with the scientific community.
We expect that credit will be given where credit is due, including in scientific writing.
Plagiarism is not tolerated.

> While a full enumeration of ethical considerations is outside of the scope of this document, CU provides a [https://www.cu.edu/ope/aps/2027] that we recommend.

Reviewer: ```suggestion
While a full enumeration of ethical considerations is outside of the scope of this document, CU provides a [document](https://www.cu.edu/ope/aps/2027) that we recommend.

```

    3. Who, if anyone, is blocking them?
    4. Who, if anyone, are they blocking?

**Lab Meeting:** 
- Lab meetings are scheduled for one hour on Wednesdays. All members of the Greene lab are expected to attend if possible. Meetings are expected to be a supportive environment for learning, constructive criticism, help, and scientific discussions. 

>    **Meeting Agenda:**  The format of each meeting will be chosen by the lab meeting lead from the options below. The lead will rotate among lab members (see below) within the group. Guests with aligned research interests may join with a supermajority vote of lab members (>⅔) and are expected to attend and participate fully.

Reviewer: ```suggestion
   **Meeting Agenda:**  The format of each meeting will be chosen by the lab meeting lead from the options below. The lead will rotate among lab members (see below) within the group. Guests with aligned research interests may join with a supermajority vote of lab members (>2/3) and are expected to attend and participate fully.

```

   - **Format 6: Preprint Review**
       - Pre-print is discussed by the group. The discussion is led by the meeting leader,  and all members are expected to be familiar with the content. The review is written collaboratively, but another member (not the meeting leader) formats, formalizes and uploads to the pre-print server as a comment.

  **Lab Meeting Lead:**  A member of the Greene lab is expected to present/lead each lab meeting as scheduled in the lab meeting calendar (found [https://docs.google.com/document/d/12mcW_1PDqmoli6W-1Mn9o0bCNHOl4Ym0yUzp2NFxBlM/edit])
   - Each member is expected to sign up as lead by the end of the previous academic semester (e.g., sign up by December for the Spring semester, sign up by June for the Fall semester.)
   - Lab meeting sign-ups will happen twice a year. Sign-ups will open in December and June for the upcoming 6 months.
   - Each member will sign up as lead for at least 2 lab meetings each semester (6 months). At least one of these meetings is expected to be a Braintrust meeting.

  **Ad-Hoc Meetings:** After meetings have been scheduled for the semester, any member can add an ad hoc meeting as needed.

>    - Ad hoc meetings are meant to help lab members get advice and help on projects, prepare for talks, oral exams, etc.

Reviewer: ```suggestion
   - Ad hoc meetings are meant to help lab members get advice and help on projects, prepare for talks, oral exams, etc.

**Individual Meetings:** We schedule weekly individual meetings.
 Once you join the lab, contact Casey and Michelle to set up a time.
 These are set up for a term to accommodate class schedules.
 We don’t reschedule these meetings by default if one of the parties (Casey or you) are out of town, so if you do want to meet in a week but travel conflicts, contact Casey and Michelle to reschedule.
 The goal of the weekly meeting is to:

 1.  Discuss challenges.
 2.  Plan strategy (project related, personal career, etc).
```
I think we lost the 1:1 meeting item
